### DISCUSSION TITLE
Creating harness drawing additional power

### DISCUSSION THREAD

**Discussion by eshroom** [revk/xESP32-Faikout#168] [General]
What are the power requirements of Faikin vs an official wifi controller, and would it be feasible to draw additional power to power a usb device. When making the harness I am considering also making a USB port drawing power from X35 in order to add usb zigbee extenders at the same time. Plug sockets, my current solution, are ok until someone unplugs them.

**Comment by revk** [comment]
No idea what power is available, and Faikin runs at 3.3V so not USB levels. It can draw around 1.6W peak when WiFi Tx I think, but best to check the ESP32 spec.

**Reply by eshroom** [reply]
Thanks. The Daikin controllers are 14v and 100ma. That gives a turn ratio of 4.24 when stepping down to 3.3v. If Faikin peak is 1.6w and 3.3v that's a draw of 485ma which is already a bit of a higher load than the Daikin controller. So probably prudent for me not to add additional load, unless the X35 port is also designed to also run high power accessories than the wifi controller.

Edit: Daikin tech support are very responsive. X35A and X39A can be used interchangeably, so there is a second ancillary power supply (at least on my PCB) that is unused by me and strong enough for some USB devices.
